src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        return false;
+      }
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand syntax for the `reset` method as well.